### PR TITLE
[ETFE-3426] demo comments

### PR DESCRIPTION
**Pull request by james-work-account**
- Aligned the items rules to the latest copydeck
- Updated h1
- Hidden the "Clear" button when JS is disabled

### CODE REVIEW

**Comment by james-work-account** [comment]
Ready to be reviewed but please don't merge if approved - want to demo locally to Lexi tomorrow before merging in case anything has been missed.

**Review by MJCallahanPage — APPROVED** [review]

**Pull request merged**
